tools/ports/__init__.py

```diff
@@ -88,14 +105,14 @@ def read_ports():
     if not filename.endswith('.py') or filename == '__init__.py':
       continue
     filename = os.path.splitext(filename)[0]
-    load_port(filename)
+    load_port_by_name(filename)
 
   contrib_dir = os.path.join(ports_dir, 'contrib')
   for filename in os.listdir(contrib_dir):
     if not filename.endswith('.py') or filename == '__init__.py':
       continue
     filename = os.path.splitext(filename)[0]
-    load_port('contrib.' + filename)
+    load_port_by_name('contrib.' + filename)
```

Comment:
The main reason is that I actually made the change locally and so I know it works fine. But when I looked at the end result, I figured (from my past PRs in this project) that it would be better to limit the amount of changes. I propose I will be submitting another PR with just these changes once this one is completed. That way we can keep this one simpler and have another less complicated one.